http2_general/test_h2_responses.py

```diff
@@ -114,3 +120,10 @@ def test_h2_bad_forwarded_for_ip(self):
 
         # perform request with invalid X-Forwarded-For header
         self.__test_h2_response(curl, "X-Forwarded-For", "1.1.1.1.1.1", http.HTTPStatus.BAD_REQUEST)
+
+    def test_h2_duplicated_content_encoding(self):
```

Comment:
`t_malformed.test_h2_malformed_headers.H2MalformedRequestsWithoutStrictParsingTest.test_content_encoding` checks `content-encoding` header using [DeproxyAutoParser](https://github.com/tempesta-tech/tempesta-test/blob/5e53cc38c03e3df22330c002bcfc00028eaf09e9/framework/deproxy_auto_parser.py#L22). You can run this test with `-v 4` and you will see the log. I think it is a duplicate test.

Reply:
I checked the code of the test case you mentioned, and I found it does not try to do matching between the client provided header and serve received header, and the execution output of that test case also prove my thoughts, which only said `Deproxy: AutoParser: Received request is valid.`. Could you provide the output in your env?

Reply:
It is checked [here](https://github.com/tempesta-tech/tempesta-test/blob/5e53cc38c03e3df22330c002bcfc00028eaf09e9/framework/deproxy_auto_parser.py#L68) and [here](https://github.com/tempesta-tech/tempesta-test/blob/5e53cc38c03e3df22330c002bcfc00028eaf09e9/helpers/deproxy.py#L304). log with `-v 6`:
```
        Deproxy: AutoParser: Received request:
GET / HTTP/1.1
host: localhost
x-forwarded-for: 127.0.0.1
content-encoding: invalid
via: 1.1 tempesta_fw (Tempesta FW 0.7.0)


        Deproxy: AutoParser: Expected request:
GET / HTTP/1.1
host: localhost
content-encoding: invalid
Via: 1.1 tempesta_fw (Tempesta FW 0.7.0)
X-Forwarded-For: 127.0.0.1


        Deproxy: AutoParser: Received request is valid.

```